packages/downloads/src/hub-request-dataset-export.ts

```diff
@@ -0,0 +1,69 @@
+import { RemoteServerError } from './remote-server-error';
+import { IHubDatasetExportRequestParameters } from './hub-dataset-export-request-parameters';
+
+/**
+ * ```js
+ * import { hubRequestDatasetExport } from "@esri/hub-downloads";
+ * 
+ * const params = {
+ *   host: 'https://hub.arcgis.com,
+ *   datasetId: 'abcdef0123456789abcdef0123456789_0',
+ *   format: 'csv'
+ * };
+ * hubRequestDatasetExport(params)
+ *   .then(response => {
+ *     // {
+ *     //   downloadId: 'abcdef0123456789abcdef0123456789_0::csv',
+ *     // }
+ *   });
+ * ```
+ * Request an export of a dataset to a particular file format.
+ * @param params - parameters defining a dataset export job
+ */
+export function hubRequestDatasetExport (params: IHubDatasetExportRequestParameters) {
```

Comment:
Are we currently blocked from downloading private content? if not, I'd expect this to take some auth info